core/src/main/java/tachyon/underfs/local/SingleMachineLocalUnderFileSystem.java

```diff
@@ -33,11 +35,7 @@
  * 
  * This only works for single machine. It is for local unit test and single machine mode.
  */
-public class UnderFileSystemSingleLocal extends UnderFileSystem {
-
-  public static UnderFileSystem getClient() {
-    return new UnderFileSystemSingleLocal();
-  }
+public class SingleMachineLocalUnderFileSystem extends UnderFileSystem {
```

Comment:
Just because it was originally had `Single` in its name, however I agree that `LocalUnderFileSystem` is a more sensible name